app/src/main/java/org/p2p/wallet/solend/ui/earn/SolendEarnFragment.kt

```diff
@@ -65,6 +66,13 @@ class SolendEarnFragment :
         }
     }
 
+    override fun setRatesErrorVisibility(isVisible: Boolean) {
+        binding.root.post {
+            TransitionManager.beginDelayedTransition(binding.root)
```

Comment:
Should be set after the visibility change.
The animation will not freeze then and will be more smooth